python's test_timer flacky

https://github.com/slint-ui/slint/actions/runs/9641761680/job/26588109691
https://github.com/slint-ui/slint/actions/runs/9675010479/job/26691717461

```
=================================== FAILURES ===================================
__________________________________ test_timer __________________________________

    def test_timer():
        global counter
        counter = 0
        def quit_after_two_invocations():
            global counter
            counter = counter + 1
            if counter >= 2:
                native.quit_event_loop()
    
        test_timer = native.Timer()
        test_timer.start(native.TimerMode.Repeated, timedelta(milliseconds=100), quit_after_two_invocations)
        native.run_event_loop()
        test_timer.stop()
>       assert(counter == 2)
E       assert 3 == 2

tests/test_timers.py:22: AssertionError
=========================== short test summary info ============================
FAILED tests/test_timers.py::test_timer - assert 3 == 2
======================== 1 failed, 24 passed in 10.60s =========================
```

--- a/api/python/tests/test_timers.py
+++ b/api/python/tests/test_timers.py
@@ -11,8 +11,8 @@ def test_timer():
     counter = 0
     def quit_after_two_invocations():
         global counter
-        counter = counter + 1
-        if counter >= 2:
+        counter = min(counter + 1, 2)
+        if counter == 2:
             native.quit_event_loop()
 
     test_timer = native.Timer()        

